dali/python/nvidia/dali/ops/_operators/python_function.pyi

```diff
@@ -100,14 +99,13 @@ class DLTensorPythonFunction:
         /,
         *input: DataNode,
         function: Optional[Callable[..., Union[Any, Sequence[Any]]]] = None,
-        batch_processing: Optional[bool] = False,
-        num_outputs: Optional[int] = 1,
+        batch_processing: bool = False,
```

Comment:
Shouldn't this be True too?

Reply:
I removed those arguments, as I don't think they will work correctly.
We need to consider revisiting the custom implementations for ops, as they do not follow the "pass any argument anywhere" flow.